src/Config/Config.php

```diff
@@ -20,12 +29,22 @@ final class Config
 
     private Finder $finder;
 
-    public function __construct(string $name = 'Default')
+    private ?string $cacheFile = '.twig-cs-fixer.cache';
```

Comment:
```suggestion
    private string $cacheFile = '.twig-cs-fixer.cache';
```
Since we set a default value